dogu.json

```diff
@@ -15,12 +15,13 @@
   "Image": "registry.cloudogu.com/k8s/nginx-ingress",
   "Dependencies": [
     {
-      "type": "dogu",
-      "name": "registrator"
+      "type": "client",
+      "name": "k8s-dogu-operator"
     },
     {
       "type": "client",
-      "name": "k8s-dogu-operator"
+      "name": "cesapp",
```

Comment:
Nice

Reply:
I improved it further with version < 0.0.0

Reply:
Done